### ISSUE TITLE
New Error

### ISSUE THREAD

**Issue by black296tuuk** [greghesp/assistant-relay#278]
I had this working for about a week and now I am getting the following error. 

Auth error:TypeError: Cannot read property 'access_token' of undefined
0|www  | Error: 14 UNAVAILABLE: Getting metadata from plugin failed with error: Cannot read property 'access_token' of undefined

I have rolled back to a backup from a week ago that I know was working and still receive the same error. 

I tried deleting the user and adding it back and the error did not change.

**Comment by ryancasler** [comment]
Are you using this as an add-on for home assistant?  This usually happens when the ownership of the token is being affected by some other software.  Are you running inside of a docker container?  If you are keeping the token outside of your container, you have to make sure that the file is accessible by the user you are running the container as.

**Comment by black296tuuk** [comment]
Yes, I am using it as an add on. My Home assistant is a VM that is hosted
on my Unraid server. I have verified that the voice command still works
directly but the test in the setup page no longer works with just a single
red X at the top.

**Comment by ryancasler** [comment]
I don't use home-assistant so I can't provide any assistance for you.  Have you tried asking the developer of the add-on?  This repo is the stand-alone GAR module.
